src/R/Components/Impl/ConnectionManager/Implementation/ViewModel/ConnectionStatusBaseViewModel.cs

```diff
@@ -30,19 +30,13 @@ internal abstract class ConnectionStatusBaseViewModel : BindableBase, IDisposabl
             IsRunning = connectionManager.IsRunning;
         }
 
-        public virtual void Dispose(bool disposing) {
-            _disposableBag.TryDispose();
-        }
-
-        public void Dispose() {
-            Dispose(true);
-        }
+        public virtual void Dispose(bool disposing) => _disposableBag.TryDispose();
```

Comment:
I know, and it is wrong. This rule assumes that derived class may have both managed and unmanaged resources, which is:
- Wrong by design, and .net framework itself follows the rule that only single unmanaged resource should be wrapped into a managed for at least 10 years (look at any `*SafeHandle` type)
- Won't happen in our application, because we don't acquire unmanaged handles in ViewModel

Can we just disable this rule?